app/helpers/service_helper.rb

```diff
@@ -15,6 +15,14 @@ def service_name
     end
   end
 
+  def update_type
+    if TradeTariffAdmin::ServiceChooser.uk?
+      'CDS'
+    else
+      'Taric'
+    end
+  end
+
```

Comment:
What about having it in the `ServiceChooser`: 

`TradeTariffAdmin::ServiceChooser.type?`

Reply:
This is only presentational/for the view